benchmarks/distributed/rpc/ps/README.md

```diff
@@ -0,0 +1,62 @@
+# RPC PS Benchmark
+
+## How to add your experiment
```

Comment:
This can be done at a later time. Perhaps creating a placeholder experiment and a quickstart of what to change in the placeholder experiment could better illustrate the steps needed

Reply:
Okay, Ill make a note to do this later.